testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -398,6 +402,9 @@ impl SubmissionWorker {
                     self.stats
                         .expired
                         .fetch_add(uncommitted.len() as u64, Ordering::Relaxed);
+                    self.stats
```

Comment:
You don't need to count latency for uncommitted transactions

Reply:
I think for this part, we count committed tx by num_requests - uncommitted.len(), so I assume we still need to count the latency for batch of txs with part of uncommitted? Or just as you said, we totally do not count latency if batch of txs just have partial commited?

Reply:
You can count latency for committed transactions only, and then calculate average as `total_committed_txn_time / num_committed_txn`

Reply:
for line #401, we also add up committed txn inside this if condition, do we really want to ignore the latency occurred by those committed txn?

Reply:
Oh, ok, you are actually right :)
Ok, I think for this first diff we can just include uncommitted txn's and count them too in latency.

When we get to p90, we probably will need to refactor `wait_for_accounts_sequence` to account for every transaction time separately, and then we can fix this to count only committed transactions

Reply:
ok, sounds good, I am going to update the pr. Thx @andll